cmd/master/main.go

```diff
@@ -4,7 +4,7 @@ import "fmt"
 import "github.com/aaronang/cong-the-ripper/master"
 
 func main() {
-	fmt.Println("master starting...")
+	fmt.Println("Starting master...")
```

Comment:
I guess grammatically the change is better..

but on topic: do we use environment variables or command line arguments